tests/basic/storage_pool_test.go

```diff
@@ -6315,3 +6315,88 @@ var _ = Describe("{PoolResizeInvalidPoolID}", func() {
 		AfterEachTest(contexts, testrailID, runID)
 	})
 })
+
+var _ = Describe("{PoolResizeReduceErrorcheck}", func() {
+	var testrailID = 79487
+	// Testrail Description : Resize with invalid pool size less than existing
+	// Testrail Corresponds : https://portworx.testrail.net/index.php?/cases/view/84470
+	var runID int
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("PoolResizeReduceErrorcheck",
+			"Resize with invalid reduced pool size",
+			nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+
+	var contexts []*scheduler.Context
+	stepLog := "Resize with reduce size than existing"
+	It(stepLog, func() {
+		log.InfoD(stepLog)
+
+		startTime := time.Now()
+
+		// Get alerts from ten hours before current time till current time to get the sufficient alerts
+		startMinusTenHours := startTime.Add(time.Duration(-600) * time.Minute)
+		endTime := time.Now()
+		alertsBefore, err := Inst().V.GetAlertsUsingResourceTypeByTime(api.ResourceType_RESOURCE_TYPE_POOL, startMinusTenHours, endTime)
+		log.FailOnError(err, "Failed to fetch alerts between startTime [%v] and endTime [%v]", startMinusTenHours, endTime)
```

Comment:
please check if you need to check for alerts ? if not please delete the lines

Reply:
removed